src/CloudFoundry.VisualStudio/ProjectPush/PublishProfileEditorResources.cs

```diff
@@ -232,6 +232,13 @@ private void EnterRefresh()
 
         private void ExitRefresh()
         {
+            if (string.IsNullOrWhiteSpace(this.PublishProfile.Organization))
```

Comment:
This doesn't look ok.
Should not pick default every time refresh is exited. 
This code should be in [RefreshClient](https://github.com/cloudfoundry-incubator/cf-vs-extension/blob/develop/src/CloudFoundry.VisualStudio/ProjectPush/PublishProfileEditorResources.cs#L309)